.eslintrc.cjs

```diff
@@ -0,0 +1,108 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2024: true,
```

Comment:
The `es2024` environment is not a valid option. You might want to use `es2021` or another valid ECMAScript version.